requirements.in

```diff
@@ -4,6 +4,7 @@ alembic==1.0.0
 Babel==2.3.4
 bleach==3.0.2
 click==6.7
+Django==1.11.21
```

Comment:
this seems like a pretty big new dependency to add, are Django's feeds really the best option?

Reply:
i'm not big fan to add django as dependency

Reply:
i'm not big fan to add django as dependency

Reply:
I knew that would have been a point of discussion :-)

If we want not to use Django utils and contrib function for feeds, truncation, safe escaping, and things like that, we would need to develop other code and rely on other libraries. The methods and classes for feeds seem very close 1:1 in terms of naming and functionalities.

In my personal opinion, the most important thing is that CKAN is able to work with Python 3 properly as soon as possible, so I would go for Django at the moment; after everything else has been done we can work on that and remove it if we want. It is a matter of priority and time. I would prefer to have this PR ready to merge soon and move to another one, instead of not having enough time to help with other issues and using my time to reimplements functions that already exist.

Reply:
> i'm not big fan to add django as dependency

Me neither for a project that is not based on Django, but it just a matter of few MBs, there is no other disadvantage. The alternative is spending more time and resources to mimic something that is already well working, tested and maintained. It is a matter of priorities. Personally I do not care if I have <10 MBs, but get a Python 3 compatible CKAN sooner. We are not even developing some new code to use Django, it is mostly a drop-in replacement. Later we can optimize as much as we want :-)